openedx/features/course_duration_limits/access.py

```diff
@@ -43,25 +51,32 @@ def get_user_course_expiration_date(user, course):
     """
     Return course expiration date for given user course pair.
     Return None if the course does not expire.
+    Defaults to MIN_DURATION.
```

Comment:
It may be valuable to either add a shortened version of the business logic from the ticket here, or to add it as comments throughout this function